climada/util/interpolation.py

```diff
@@ -59,7 +60,8 @@ def dist_sqr_approx(lats1, lons1, cos_lats1, lats2, lons2):
     return d_lon * d_lon * cos_lats1 * cos_lats1 + d_lat * d_lat
 
 def interpol_index(centroids, coordinates, method=METHOD[0],
-                   distance=DIST_DEF[1], threshold=THRESHOLD):
+                   distance=DIST_DEF[1], threshold=THRESHOLD,
```

Comment:
```suggestion
                   distance=DIST_DEF[2], threshold=THRESHOLD,
```

Reply:
Shall we make the 'euclidian' distance the default as discussed before?

Reply:
Yes, let's do it. But please mention in the docstring below that `'euclidean'` is the default and also mention that there is a slower but more accurate alternative `'haversine'`.

Reply:
Good point, will do! Thanks.

Reply:
I now added docstrings to explain the default use of 'euclidian' and that 'haversine' can be used when higher accuracy is needed. I hope it is clear 
3fb7778